schemas/user.py

```
@@ -0,0 +1,61 @@
+        #한글,염문,숫자만 허용하고 공백이나 특수문자는 안됨
+        pattern=r"^[가-힣a-zA-Z0-9]+$",
+        description="닉네임 (2~10자, 특수문자 불가)"
+    )
+
+#회원가입 요청
+class UserCreate(UserBase):
+    password: str = Field(...,min_length=8,max_length=20,description="비밀번호 (영문/숫자/특수문자 포함)")
+
+    @field_validator("password") # 클래스 내부의 필드 password
+    @classmethod # 아직 객체가 생성되지 않았으므로 클래스메서드 선언
+    def validate_password(cls,v):
```

Ruff (0.14.11)

Missing return type annotation for classmethod `validate_password`

(ANN206)